tests/fixtures/sort/multiple_decimals_numeric.expected

```diff
@@ -0,0 +1,35 @@
+-2028789030
+-896689
+-8.90880
+-1
+-.05
+
+
+
+
+
+
+
+
+000
+CARAvan
+00000001
+1
+1.040000000
+1.444
+1.58590
+8.013
+45
+46.89
+										4567..457
```

Comment:
@electricboogie (since you added this test, I just moved the expected result to its own file) I changed the expected result of this test because we're now parsing `4567..457` as `4567` because we stop when we encounter the second decimal point. Previously it was parsed as `4567.457` because of `remove_trailing_dec`. I think the new behavior is actually more in line with GNU sort, but I was having trouble finding a way to get GNU sort to generate this result. @electricboogie do you remember which locale you used to generate this test result? `LC_ALL=C` seems to be closest, but it does not accept thousands separators. `en_US.utf-8` in turn does support thousands separators, but uses a slightly different last resort comparison (I think it trims before comparing).

Reply:
I don't pass any env vars myself.  Therefore, would have to be my LANG=en_US.UTF-8?

Reply:
pkgsrc packaged gsort on MacOS -- sort (GNU coreutils) 8.32

It's also possible I flubbed and used the Apple/BSD sort on accident, but this would be the first time I've seen them give different output.

Reply:
No, you're totally right not sure why I thought en_US.utf-8 did not match. It produces exactly the output we had before. Sorry for the confusion